rules/import-index.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+const isImportingIndex = m => m.endsWith('/') || m.endsWith('index.js') || m.endsWith('index');
```

Comment:
`index.js` => `/index.js` and `index` => `/index`, otherwise something `require('./lindex.js')` will trigger it. Why not just reuse the regex below?